diff -urN ddt-0.5.9.org/Makefile.am ddt-0.5.9/Makefile.am
--- ddt-0.5.9.org/Makefile.am	Tue May  6 13:10:39 2003
+++ ddt-0.5.9/Makefile.am	Tue May  6 13:24:37 2003
@@ -13,13 +13,9 @@
 endif
 
 if DDT_DOCS
-DOCS_DIR += docs
+DOCS_DIR = docs
 endif
 
 SUBDIRS = ${BASEDIRS} ${CLIENT_DIR} ${SERVER_DIR} ${ADMIN_DIR} ${DOCS_DIR}
 
-if DDT_DEBUG
-CXXFLAGS += -g
-CFLAGS += _DEBUG
-endif
 
diff -urN ddt-0.5.9.org/admin/Makefile.am ddt-0.5.9/admin/Makefile.am
--- ddt-0.5.9.org/admin/Makefile.am	Tue May  6 13:10:39 2003
+++ ddt-0.5.9/admin/Makefile.am	Tue May  6 13:17:31 2003
@@ -1,7 +1,7 @@
 CXXFLAGS = -g -O2 -fno-strict-prototype
 INCLUDES = -I../framework/include @PGSQL_INCLUDE@ @BIND_INCLUDE@
 LDFLAGS  = -L../framework/lib @PGSQL_LDFLAGS@ @BIND_LDFLAGS@ @OPT_LDFLAGS@
-LDADD   += -lLogger -lDdtDB @PGSQL_LIBS@ @BIND_LIBS@ @OPT_LIBS@
+LDADD   = -lLogger -lDdtDB @PGSQL_LIBS@ @BIND_LIBS@ @OPT_LIBS@
 
 noinst_PROGRAMS = ddt-admin admin.cgi
 
diff -urN ddt-0.5.9.org/configure.in ddt-0.5.9/configure.in
--- ddt-0.5.9.org/configure.in	Tue May  6 13:10:39 2003
+++ ddt-0.5.9/configure.in	Tue May  6 13:18:51 2003
@@ -93,6 +93,8 @@
   AC_DEFINE(DDT_DEBUG)
 fi
 
+AC_CHECK_LIB(tinfo, tgetflag, , AC_MSG_ERROR(tinfo lib not found!))
+
 dnl Use --bindir to specify location of user binaries.
 dnl Use --sbindir to specify location of system binaries.
 dnl Use --mandir to specify location of manpages.
