diff -urN ddt-0.5.9.org/configure.in ddt-0.5.9/configure.in
--- ddt-0.5.9.org/configure.in	Tue May  6 14:39:02 2003
+++ ddt-0.5.9/configure.in	Tue May  6 14:39:17 2003
@@ -263,7 +263,7 @@
 					  AC_MSG_ERROR(gcrypt library not found))
 
 if test x$admin = xtrue ; then
-  AC_CHECK_LIB(cgi, cgiHeader, ,      AC_MSG_ERROR(cgi library not found))
+  AC_CHECK_LIB(cgic, cgiHeader, ,      AC_MSG_ERROR(cgi library not found))
 fi
 
 AC_LANG_RESTORE

