diff -urN ddt-0.5.org/Makefile.am ddt-0.5/Makefile.am
--- ddt-0.5.org/Makefile.am	Wed Sep 18 14:41:30 2002
+++ ddt-0.5/Makefile.am	Wed Sep 18 14:42:53 2002
@@ -9,12 +9,12 @@
 endif
 
 if DDT_DEBUG
-CXXFLAGS += -g
-CFLAGS += _DEBUG
+CXXFLAGS = -g
+CFLAGS = _DEBUG
 endif
 
 if DDT_DOCS
-DOCS_DIR += docs
+DOCS_DIR = docs
 endif
 
 SUBDIRS = ${BASEDIRS} ${SERVER_DIR} ${ADMIN_DIR} ${DOCS_DIR}
diff -urN ddt-0.5.org/acinclude.m4 ddt-0.5/acinclude.m4
--- ddt-0.5.org/acinclude.m4	Wed Sep 18 14:41:30 2002
+++ ddt-0.5/acinclude.m4	Wed Sep 18 14:59:05 2002
@@ -106,7 +106,7 @@
     fi
     
     if test "x$BIND_LIBS" = "x"; then
-      BIND_LIBS=-lbind
+      BIND_LIBS=-lresolv
     fi
 
     old_CFLAGS=$CFLAGS
@@ -116,7 +116,7 @@
     LDFLAGS="$LDFLAGS $BIND_LDFLAGS"
     LIBS="$LIBS $BIND_LIBS"
     
-    AC_CHECK_LIB(bind,__res_update,[
+    AC_CHECK_LIB(resolv,__res_update,[
         AC_SUBST(BIND_LIBS)
         AC_SUBST(BIND_LDFLAGS)
         AC_SUBST(BIND_INCLUDE)
@@ -143,7 +143,7 @@
     fi
 
     BIND_LDFLAGS=-L$BIND_LIBDIR
-    BIND_LIBS=-lbind
+    BIND_LIBS=-lresolv
     ])
     
     AC_ARG_WITH(bind-include,
diff -urN ddt-0.5.org/admin/Makefile.am ddt-0.5/admin/Makefile.am
--- ddt-0.5.org/admin/Makefile.am	Wed Sep 18 14:41:30 2002
+++ ddt-0.5/admin/Makefile.am	Wed Sep 18 14:43:06 2002
@@ -1,7 +1,7 @@
 CXXFLAGS = -g -O2 -fno-strict-prototype
 INCLUDES = -I../framework/include @PGSQL_INCLUDE@ @BIND_INCLUDE@
 LDFLAGS  = -L../framework/lib @PGSQL_LDFLAGS@ @BIND_LDFLAGS@ @OPT_LDFLAGS@
-LDADD   += -lLogger -lDdtDB @PGSQL_LIBS@ @BIND_LIBS@ @OPT_LIBS@
+LDADD   = -lLogger -lDdtDB @PGSQL_LIBS@ @BIND_LIBS@ @OPT_LIBS@
 
 noinst_PROGRAMS = ddt-admin admin.cgi
 
diff -urN ddt-0.5.org/configure.in ddt-0.5/configure.in
--- ddt-0.5.org/configure.in	Wed Sep 18 14:41:30 2002
+++ ddt-0.5/configure.in	Wed Sep 18 14:47:29 2002
@@ -67,6 +67,8 @@
 dnl AC_CHECK_LIB(opt, main, , AC_MSG_ERROR(opt lib not found!)) 
 AC_LIB_OPT
 
+AC_CHECK_LIB(tinfo, tgetflag, , AC_MSG_ERROR(tinfo lib not found!))
+
 AC_DEFINE(CONF_DIR, "@sysconfdir@")
 AC_SUBST(sysconfdir)
 
